### ISSUE TITLE
multiple paragraphs

### ISSUE THREAD

**Issue by th-km** [yoanmalie/kirbytextWrap#1]
The plugin looks really useful, espacially for utility and component classes. I'm wondering if there is a way to add support for mulitple paragraphs styling? So far, it only returns the first paragraph.

Ideally something like this `$page->title()->ktw('p', ['class' => 'custom'])` would returns:
```php
<p class="custom">...</p>
<p class="custom">...</p>
<p class="custom">...</p>
...
```

**Comment by th-km** [comment]
@yoanmalie any update on this?

**Comment by yoanmalie** [comment]
@th-km  Hi, sorry for the lack of reply, I was on vacation.

Thank for your suggestion, that's I totally agree. I will give a look on how to do it properly. I have some questions to clear out, like sometime we only have to change the first one, or sometime all (that's what you want). So we maybe need another one parameter…

Since I'm not a PHP developer, I can't tell you that I will be done asap.
To anyone who's read this, feel free to help me on this.

By the way, I'm aware of this PR https://github.com/getkirby/kirby/pull/1442 and I will probably have some breaking changes to do in the future depending on what and how they'll do it exactly.

**Comment by th-km** [comment]
Yeah I know, It's has always been difficult for me to style markdown and avoid things like this in my css:
```css
p + p {
margin-top: 1.5rem;
}
```
I use `toStructure()`a lot without any decent alternative.

It'd be great to have acces to mutilple vars instead of one, basically all tags provided by markdown, with a default to null if nothing is specified. This will allow to style every tags the way you want (BEM, utility and so on).
